youtube_dl/extractor/bandlab.py

```diff
@@ -0,0 +1,191 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+
+class BandlabIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?bandlab\.com/post/(?P<id>[^/]+)'
+
+    _TEST = {
+        'url': 'https://www.bandlab.com/post/f5f04998635a44ea819cacdba7ae2076_f8d8574c3bdaec11b6562818783151a1',
+        'info_dict': {
+            'id': 'f5f04998635a44ea819cacdba7ae2076_f8d8574c3bdaec11b6562818783151a1',
+            'ext': 'm4a',
+            'title': 'ON MY OWN (unreleased)',
+            'artist': 'Michael MacDonald',
+        },
+    }
+
+    def _real_extract(self, url):
+        track_id = self._match_id(url)
+        config = self._download_json(
+            'http://www.bandlab.com/api/v1.3/posts/%s' % track_id, track_id)
+        track_url = config['track']['sample']['audioUrl']
+
+        return {
+            'id': track_id,
+            'title': config['track']['name'],
```

Comment:
Pull out mandatory field:
```suggestion
        title = config['track']['name']

        return {
            'id': track_id,
            'title': title,
```

Reply:
Also this. At the moment `title` is a required field, so it should be set on a line by itself in case there's no `name` in `config['track']`.